### DISCUSSION TITLE
Chinese render error when use Geodataframe.plot

### DISCUSSION THREAD

**Discussion by forestbat** [geopandas/geopandas#3414] [Q&A] [answered]
I use code below to render Chinese in JupyterLab:
```
fig, ax = plt.subplots(figsize=(10, 10))
# stations is a Geodataframe and STNM is its column with Chinese
for idx, row in stations.iterrows():
    # font_prop = FontProperties(fname='./.conda/envs/torchhydro1/fonts/Ubuntu-Th.ttf')
    # UserWarning: Glyph (\N{CJK UNIFIED IDEOGRAPH-53F0}) missing from font(s) ……
    ax.text(row.geometry.x, row.geometry.y, row['STNM'], fontsize=9, ha='center', va='bottom', color='red', font_properties=font_prop)
```
But even if I has tried all methods in [this topic](https://stackoverflow.com/questions/75993481/matplotlib-plot-with-chinese-label-userwarning-glyph-25293-ncjk-unified-ide), result is still wrong:
![图片](https://github.com/user-attachments/assets/8351d8ef-1ea8-453f-bcc2-d8e14168c690)
So how to solve the problem?

**Comment by martinfleis** [comment]
As the warning suggests, you should use a font that supports glyphs you are trying to plot.

**Reply by forestbat** [reply]
I have tried`Dejavu Sans`, `SimHei`, `Arial` and `SimSun-ExtB` and so on, but they are all failed.

**Reply by martinfleis** [reply]
I got no insight into Chinese glyphs, I'm sorry. This is ultimately an issue with maptlotlib not geopandas, so I'd suggest asking there.

**Reply by forestbat** [reply, marked as answer]
I have run commands below:
```
 fc-cache -f -v
 fc-list :lang=zh
/usr/share/fonts/truetype/droid/DroidSansFallbackFull.ttf: Droid Sans Fallback:style=Regular
```
Then I used `plt.rcParams['font.family'] = 'Droid Sans Fallback'` and solved this problem.
I need to run unit grid for serveral times to make it become normally.
